python/lsst/ci/prepare.py

```diff
@@ -499,6 +507,9 @@ def do_fetch_products(self, products: list[str], refs: list[str]):
             loop.run_until_complete(self.resolve_versions())
         if not self.no_fetch and len(self.lfs_product_names):
             loop.run_until_complete(self.lfs_checkout())
+
+        # Calling list_non_default_refs_prs method
```

Comment:
Consider different phrasing here because the next line is calling exactly that method so no need to duplicate the method name.